src/js/helpers/sale-helper.js

```diff
@@ -21,17 +21,26 @@ export async function createSaleDescriptionBlobId (description) {
   return blob.id
 }
 
-export async function createBalanceIfNotExist (assetCode) {
-  let accountBalancesAssetsCodes =
+export async function createBalanceIfNotExist (...assetsCodes) {
```

Comment:
remove this function, add this logic to vuex types.CREATE_BALANCE, and use from vuex